image_manifests.json

```diff
@@ -59,10 +59,15 @@
             }
         }
     },
-    "ghcr.io/hannes-ucsc/bigquery-emulator:azul": {
-        "digest": "sha256:5500114fdbe7227e5a05571a5a1df21ca94625c9fef1ae22b721e39c8ea06c8c",
-        "id": "sha256:2fc698b738dd0feeab79fa175dbca3dc4cf59f7a766e16b85541327e6b3681fc",
-        "platform": "linux/amd64"
+    "docker.io/ucscgi/azul-bigquery-emulator:0.4.4-1": {
+        "digest": "sha256:d522400b7ca566cec2deb49fa4302936990d24b39333530c189921d921c0d6d6",
+        "manifests": {
```

Comment:
Technically, there is no point in having a singleton index image. As we learned the hard way, it actually breaks certain scenarios.  No other image currently has only one entry in `manifests`, and this one shouldn't either. This may be the result of listing a platform explicitly in the build action of the other repository.